ginga/examples/configs/plugin_SaveImage.cfg

```diff
@@ -0,0 +1,22 @@
+#
+# SaveImage plugin preferences file
+#
+# Place this in file under ~/.ginga with the name "plugin_SaveImage.cfg"
+
+# Default output parameters. Can also be changed in the GUI.
+output_directory = '.'
+output_suffix = 'ginga'
+
+# Clobber existing output files or not
+clobber = False
+
+# Only list modified images from the channel
+modified_only = True
```

Comment:
Sorry, didn't notice that, right off that bat.  Yes, it works, but wondering whether the general (non-editing) user will be confused if the default is not `False`.